File: src/main/org/nlogo/prim/etc/_turtlecode.java
package org.nlogo.prim.etc ;

import org.nlogo.nvm.Command;
import org.nlogo.nvm.Syntax;

public final strictfp class _turtlecode
	extends Command
	implements org.nlogo.nvm.CustomAssembled
{
	@Override
	public Syntax syntax()
	{
		return Syntax.commandSyntax( "-T--" , false ) ;
	}
	@Override
	public void perform( final org.nlogo.nvm.Context context )
	{
	    throw new UnsupportedOperationException() ;
	}
	public void assemble( org.nlogo.nvm.AssemblerAssistant a )
	{
		// do nothing -- drop out of existence
	}
}

File: src/main/org/nlogo/prim/etc/_turtlecode.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.api.Syntax
import org.nlogo.nvm.{ Command, Context, CustomAssembled, AssemblerAssistant }

class _turtlecode extends Command with CustomAssembled {
  override def syntax =
    Syntax.commandSyntax("-T--", false)
  override def perform(context: Context) {
    throw new UnsupportedOperationException
  }
  def assemble(a: AssemblerAssistant) {
    // by doing nothing here, drop out of existence
  }
}
